env_dump.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Script to dump shell environment variables (loaded by CircleCI) to local
+# configuration file to be used by Next.js
+
+template_env_file=".env.template"
```

Comment:
Better to rename `.env.template` to `.env.circleci.template` or `.env.circleci-template`. Then, other developers will get a rough idea of where the env file is used for just referring to its filename.

Reply:
Ok, I just had another look, and it seems CircleCI replace the raw values with asterisks. That should be very low risk and all good.